src/components/BasketballCourt/CourtArea.tsx

```diff
@@ -15,7 +15,7 @@ const CourtArea: React.FC<CourtAreaProps> = ({ courtWidth, startPoint }) => {
 
   const selectedColor = useStoreSelector((state) => state.courtColor.selectedColor);
   const courtAreaColor = useStoreSelector(
-    (state) => state.tile.find((tile) => tile.location.includes("courtArea"))?.color
+    (state) => state.tile.court?.find((tile) => tile.location.includes("courtArea"))?.color
```

Comment:
I think you can extract this part of the logic into a reusable function, because I see you use the same method in other files, and this improves the readability of the code
<img width="830" alt="image" src="https://user-images.githubusercontent.com/61776449/176843753-d67dcbe4-d206-4341-baa5-79a138dbfee2.png">

Reply:
Thank you! Done.